src/Classes/ServiceAPI/MyRadio_Scheduler.php

```diff
@@ -185,9 +185,14 @@ public static function getScheduleConflicts($term_id, $time)
     {
         self::initDB();
         $conflicts = [];
-        $start_day = (new MyRadio_Term($term_id))->getTermStartDate() + ($time['day'] * 86400);
+        $term = new MyRadio_Term($term_id);
+        $start_day = $term->getTermStartDate() + ($time['day'] * 86400);
         //Iterate over each week
-        for ($i = 1; $i <= 10; ++$i) {
+        // Changes it to get number of weeks in a semester from Myradio_term
+
+        
```

Comment:
This comment isn't necessary - once this PR is merged, the change will have been applied, so there's no need for it (the commit message documents it enough that you don't need a comment).

Otherwise looks good to me, thank you!